e2e/global.setup.ts

```diff
@@ -0,0 +1,90 @@
+import { join as pjoin } from 'path';
+import { test as setup, expect } from '@playwright/test';
+import { goToDataSet } from './utils';
+
+export const STORAGE_STATE = pjoin(__dirname, '.auth.json');
+
+// TODO: this file is ignored, a
+setup('login and create account', async ({ page }) => {
+  const { SPLUNK_USER: user, SPLUNK_PASSWORD: password, CREATE_ACCOUNT: createAccount } = process.env;
+
+  // Login
+  await page.goto('/');
+  await page.screenshot({ path: 'playwright-screenshots/page-login.png', fullPage: true });
+
+  console.log("Fill in login form");
+  await page.getByPlaceholder('Username').fill(user || 'SPLUNK_USER env is empty');
+  await page.getByPlaceholder('Password',  { exact: true }).fill(password || 'SPLUNK_PASSWORD env is empty');
```

Comment:
Maybe instead of `SPLUNK_PASSWORD env is empty` we could check for the env values and throw an exception if they are not defined.

Reply:
Good point! In these times I wasn't soo fluent in Playwright. In that time it was good for me, since it has crashed in the next step, since it wasn't capable of logging in.

Reply:
Fixed